vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -192,6 +214,78 @@ module Tuples =
     printfn "Struct Tuple: %A\nReference tuple made from the Struct Tuple: %A" sampleStructTuple (sampleStructTuple |> convertFromStructTuple)
 
 
+/// The F# pipe operators ('|>', '<|', etc.) and F# composition operators ('>>', '<<')
+/// are used extensively when processing data.  These operators are themselves functions
+/// which make use of Partial Application.
```

Comment:
I've capitalized it throughout the document, and also have a link to the section in our docs which explains it.